docs/software/desktop.md

```diff
@@ -172,24 +172,92 @@ Website: <http://gnustep.org>
+
+    <h4>Activate Chromium kiosk mode</h4>
+
+    To put your Chromium browser into kiosk mode the following steps have to be achieved:
+
+    1. Install Chromium browser via 
+
+        ```sh
+        dietpi-software install 113
+        ```
+
+        If Chromium is not installed, you will get a startup error message that the file `/var/lib/dietpi/dietpi-software/installed/chromium-autostart.sh` is not found if you activate Chromium kiosk mode via `dietpi-autostart`.
+
+    1. Select Chromium kiosk mode (with automatic login) via
+
+        ```sh
+        dietpi-autostart
+        ```
+
+        and select `11 : Chromium - Dedicated use without desktop` in the main dialog.  
+        In the following dialog enter the autostart web page which shall be displayed initially.
+
+    1. Reboot to the Chromium kiosk mode
+
+    <h4>Deactivate Chromium kiosk mode</h4>
+
+    To deactivate the kiosk mode, e.g. login to the system via ssh and execute `dietpi-autostart` to switch away from the kiosk mode.
+
+    <h4>Tweak kiosk mode behaviour</h4>
+
+    The kiosk mode behaviour is defined by the contents of the file  
+    `/var/lib/dietpi/dietpi-software/installed/chromium-autostart.sh`  
+    which can be tweaked by editing it.
+
+    There are some hints how to achieve tweaking:
+
+    - [DietPi issue #3389: *Chromium Autostart / Kiosk Mode*](https://github.com/MichaIng/DietPi/issues/3389)
+    - [DietPi Forum entry #14886: *Non-Interactive Kiosk Improvements*](https://dietpi.com/forum/t/non-interactive-kiosk-improvements/14886)
+    - [DietPi issue #2575: *Chromium: Hide mouse in autostart kiosk mode*](https://github.com/MichaIng/DietPi/issues/2575)
+    - [DietPi Forum entry #15769: *How do I get rid of the mouse pointer when I start chromium w/o desktop?*](https://dietpi.com/forum/t/how-do-i-get-rid-of-the-mouse-pointer-when-i-start-chromium-w-o-desktop/15769)
+    - [DietPi issue #2938: *Chromium: Allow to use app mode instead of kiosk mode*](https://github.com/MichaIng/DietPi/issues/2938)
+
+=== "Enable Widevine DRM protected content"
+
+    To enable the support for Widevine DRM protected content, these options exist:
+
+    - Manually: [https://blog.vpetkov.net/2019/07/12/net ... -chromium/](https://blog.vpetkov.net/2019/07/12/netflix-and-spotify-on-a-raspberry-pi-4-with-latest-default-chromium/)
+    - with scripts: [https://blog.vpetkov.net/2020/03/30/ras ... many-others/](https://blog.vpetkov.net/2020/03/30/raspberry-pi-netflix-one-line-easy-install-along-with-hulu-amazon-prime-disney-plus-hbo-spotify-pandora-and-many-others/)
+
+=== "FAQs"
+
+    <h4>Does my device support GPU acceleration?</h4>
+
+    Not all devices support GPU acceleration with Chromium.
+
+    - RPis supports native OpenGL and will be configured during installation.
+    - Odroids support OpenGLES and will be configured during installation.
+    - Other devices will most likely be limited to software rendering.
```

Comment:
This info is outdated and fluent anyway. Most SBCs nowadays support most hardware acceleration in Chromium. We can either remove this list, or a more generic info would be that old hardware/chips may not support hardware acceleration for new graphics features (like Vulkan) yet and very new hardware/chips might not yet have support enabled in the used (mainline) kernel or userspace libraries.

Reply:
I propose to kick this FAQ entry out.

Reply:
Also fine with me.